### PR TITLE
Stanislaw nits

### PR DESCRIPTION
**Pull request by chris-wood**

### CODE REVIEW

File: draft-irtf-cfrg-voprf.md [hunk 1]
```diff
@@ -1610,11 +1632,9 @@ with the following distinguishing properties:
 one DLEQ proof object. This is enabled using an established batching technique {{DGSTV18}}.
 
 The pseudorandomness and input secrecy (and verifiability) of the OPRF (and
-VOPRF) protocols in {{JKK14}} are based on an assumption with oracle access to the
-Computational Diffie Hellman (CDH) assumption, known as the One-More Gap CDH,
-that is computationally difficult to solve in the corresponding prime-order
-group. In {{JKK14}}, these properties are proven for one instance
-(i.e., one key) of
+VOPRF) protocols in {{JKK14}} are based on the One-More Gap Computational
+Diffie Hellman (CDH) assumption that is computationally difficult to solve in the corresponding prime-order group.
```
Comment (armfazh) [inline draft-irtf-cfrg-voprf.md:1636]: This change re-introduces an error in the notation, previously pointed during a review.

Observe that the abbreviation `CDH` applies only to `Computational Diffie-Hellman` part and not to `One-More Gap Computational Diffie-Hellman`.

Comment (chris-wood) [reply draft-irtf-cfrg-voprf.md:1636]: I don't think this is an error. It's clear that CDH applies to a Computational Diffie Hellman.

Comment (chris-wood) [reply draft-irtf-cfrg-voprf.md]: I removed the abbreviation altogether.

**Comment by chris-wood** [comment]
@armfazh please approve or request changes so we can move this forward -- thanks!

**Review by armfazh — APPROVED** [review]

**Pull request merged**